### PR TITLE
Manage permissions for The Millais Alliance (Do not merge)

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: app/views/onboard/step1.njk [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{% block primary %}
+
+
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <p>Permissions control who in your organisation can see sensitive information and make decisions about applications.</p>
+
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Your organisation</h3>
```
Comment (paulrobertlloyd) [inline app/views/onboard/step1.njk:20]:
```suggestion
      <h3 class="govuk-heading-m govuk-!-margin-bottom-2">Your organisation</h3>
```

File: app/views/onboard/step1.njk [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <p>The Millais Alliance</p>
+
+      <p class="govuk-heading-s govuk-!-margin-bottom-0">Partner organisations</p>
```
Comment (paulrobertlloyd) [inline app/views/onboard/step1.njk:23]:
```suggestion
      <p class="govuk-heading-m govuk-!-margin-bottom-2">Partner organisations</p>
```

File: app/views/onboard/step2.njk [hunk 3]
```diff
@@ -1,38 +1,57 @@
...
+
+      <form novalidate action="/onboard/step3">
         {{ govukCheckboxes({
           idPrefix: "permissions",
           name: "permissions",
           fieldset: {
             legend: {
-              html: "Titan Partnership Ltd can",
+              html: "University of Brighton can",
               classes: "govuk-fieldset__legend--s"
```
Comment (paulrobertlloyd) [inline app/views/onboard/step2.njk:44]:
```suggestion
              classes: "govuk-fieldset__legend--m"
```

File: app/views/onboard/check.njk [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          </ul>
+
+          <p>The Millais Alliance can:</p>
+          <ul class="list list--bullet">
+            <li>make decisions</li>
+            <li>see safeguarding information</li>
+          </ul>
+        {% endset %}
+
+        {{ govukSummaryList({
```
Comment (paulrobertlloyd) [inline app/views/onboard/check.njk:61]: The summary list seems a bit unwieldy here. Would it work better is the content was able to stretch the full width of the page?

Screenshot 2020-04-21 at 14 41 59

Comment (adamsilver) [reply app/views/onboard/check.njk:61]: Yeah, it is, will try it out!

**Pull request merged**